MekHQ/docs/scenariomodifiers.txt

```diff
@@ -0,0 +1,16 @@
+modifierName - name of the modifier, intended to be displayed in the various modifier dropdowns. Currently not working.
+additionalBriefingText - adds extra arbitrary text to the scenario briefing. Works fine.
+benefitsPlayer - used to indicate whether this modifier is helpful to the player or not. Unused.
+blockFurtherEvents - used to indicate that no further events should be processed after this modifier. Not sure if working or not, probably not.
+eventTiming - PreForceGeneration or PostForceGeneration. Indicates when the modifier is eligible to be applied.
+forceDefinition - a ScenarioForceTemplate definition. 
+skillAdjustment - adjusts the skill rating of all the units generated up until this point up or down the indicated # of levels.
+qualityAdjustment - adjusts the equipment quality of all units generated from this point on up or down the indicated # of levels.
```

Comment:
Just checking: it is correct that the first adjusts already generated units, the second adjusts units generated after?

Reply:
That's correct.